sleap/gui/commands.py

```
@@ -3470,13 +3481,203 @@ def do_action(context: CommandContext, params: dict):
+            inds=range(*frame_range),
+            copy=True,  # Ensures a deep copy of the extracted labels
+        )
+
+        # Remap frame indices in pruned_labels to start from 0
+        for labeled_frame in pruned_labels.labeled_frames:
+            labeled_frame.frame_idx -= frame_range[0]
+
+        try:
+            pkg_filename = params["filename"]
+            pruned_labels.save(pkg_filename, with_images=True)
+        except Exception as e:
```

Ruff (0.8.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)